src/main/msp/msp.c

```diff
@@ -1344,12 +1344,14 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 1); // Rate numerator, not used anymore
         sbufWriteU8(dst, blackboxGetRateDenom());
         sbufWriteU16(dst, blackboxConfig()->p_ratio);
+        sbufWriteU8(dst, systemConfig()->debug_mode);
```

Comment:
Please add '//Added in MSP API 1.42' comment here as well.

Reply:
Added